app/(auth)/register.js

```diff
@@ -95,6 +98,44 @@ export default function Register() {
 		}
 	};
 
+		const handlePresentModalPress = useCallback(() => {
+		bottomSheetModalRef.current?.present();
+	}, []);
+
+	const handleModalPress = (
```

Comment:
honestly, this whole function is overkill for just displaying the terms. it can be deleted, and instead just call `bottomSheetModalRef.current.present()` in the onPress.